admin-dev/themes/new-theme/scss/pages/_modules.scss

```diff
@@ -611,20 +683,34 @@ form#importDropzone {
     text-align: center;
     vertical-align: middle;
     img {
-      max-width: 57px;
+      max-width: 2.85em;
     }
   }
 
-  .module-column-infos {
+  .module-column-infos,
+  .module-column-description {
     display: table-cell;
     height: 50px;
     vertical-align: middle;
+    font-size:.95em;
     .module-name {
-      font-size: 1.2em;
+      color: $primary;
     }
 
     .module-description {
-      @extend .text-muted;
+      color: #363a41;
+    }
+  }
+
+  .module-column-infos {
+    width: 200px;
+    padding: 0 4px;
+  }
+
+  .module-column-description {
+    width: 679px;
```

Comment:
Weird to have this kind of width, is it ok with different size than yours, like responsive?

Reply:
up!

Reply:
Yeah this div block only display in medium screens. See the positions.html.twig (l.132).